src/structures/VoiceChannel.js

```diff
@@ -59,6 +65,10 @@ class VoiceChannel extends BaseGuildVoiceChannel {
        */
       this.rateLimitPerUser = data.rate_limit_per_user;
     }
+
+    if ('nsfw' in data) {
+      this.nsfw = Boolean(data.nsfw);
```

Comment:
I don't understand why this wasn't removed in the other PR but this is redundant
```suggestion
      this.nsfw = data.nsfw;
```